src/services/utils/jwt.ts

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs'
+import jwt from 'jsonwebtoken'
+import { makeLogger } from 'lido-nanolib'
+
+export type JwtService = ReturnType<typeof makeJwtService>
+
+export const makeJwtService = (
+  logger: ReturnType<typeof makeLogger>,
+  { JWT_SECRET_PATH }: { JWT_SECRET_PATH: string }
+) => {
+  const generateToken = () => {
+    try {
+      // read the secret from the file
+      const hexSecret = fs.readFileSync(JWT_SECRET_PATH, 'utf8').trim()
```

Comment:
Here we're using the synchronous version of the method, but I’d suggest using the asynchronous one from fs/promises.

Reply:
done